include/kan/vfs.h

```diff
@@ -18,4 +18,16 @@ typedef struct vfs_node_s {
   struct vfs_node_s *children; /* NULL for files */
 } vfs_node_t;
 
+typedef enum {
+  O_CREATE = 0x01,
+  O_TRUNCATE = 0x02,
+  O_APPEND = 0x04,
+  O_READ = 0x08,
+  O_WRITE = 0x10,
+  O_RDWR = 0x20,
+  O_SYNC = 0x40,
+} vfs_mode_t;
```

Comment:
I wouldn't use enums, just a list of `#define`s and `short` for the arguments should be enough

Reply:
ok, i can fix that now

Reply:
![image](https://user-images.githubusercontent.com/41549137/215340311-a6263324-07d3-45bb-9a11-17328b910e1e.png)